src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,87 @@
+import { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import usersFromServer from '../../api/users';
+import todosFromServer from '../../api/todos';
+
+type Props = {
+  onAdd: (value: Todo) => void
+};
+
+export const TodoForm: React.FC<Props> = ({ onAdd }) => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+
+  function onHandle(e: React.FormEvent) {
```

Comment:
[CHECKLIST] - Don't use one-letter variable naming. (It could be confusing for another developer, who reads your code. Also, it would really hard to find this variable in the code editor using search)